src/main/resources/files/gradle/liberty/build.gradle.tpl

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+group = '[# th:text="${maven_groupid}"/]'
+version = '1.0-SNAPSHOT'
+
+description = "MicroProfile Starter example"
+
+sourceCompatibility = [# th:text="${se_version}"/]
+targetCompatibility = [# th:text="${se_version}"/]
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+// configure liberty-gradle-plugin
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.1.1'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    providedCompile 'org.eclipse.microprofile:microprofile:[# th:text="${mp_version}"/]'
+}
+
+ext  {
+    liberty.server.var.'default.http.port' = '[# th:text="${port_service}"/]'
+    liberty.server.var.'app.context.root' = '/'
```

Comment:
You need to use app.context.root in server.xml
`<webApplication id="GradleSample" location="GradleSample.war" contextRoot="${app.context.root}" />`

Reply:
It is already

`<webApplication location="${project.name}.war" contextRoot="${app.context.root}">`